### PR TITLE
add overflow check for batch write

### PR DESCRIPTION
**Pull request by marsishandsome**

### CODE REVIEW

**Comment by marsishandsome** [comment]
@birdstorm @zhexuany PTAL

File: tikv-client/src/test/java/com/pingcap/tikv/codec/TableCodecTest.java [hunk 1]
```diff
@@ -24,8 +24,8 @@ private static TiTableInfo createTable() {
         // TODO: enable when support Timestamp
         // .addColumn("c3", DateTimeType.DATETIME)
         // .addColumn("c4", TimestampType.TIMESTAMP)
-        .addColumn("c5", StringType.VARCHAR)
-        .addColumn("c6", StringType.VARCHAR)
+        .addColumn("c5", StringType.VARCHAR255)
```
Comment (zhexuany) [inline tikv-client/src/test/java/com/pingcap/tikv/codec/TableCodecTest.java]: We may change the signature of `addColumn` as the following, 

 .addColumn("c5", StringType.VARCHAR, 235)

where 235 specifying the length.

Comment (marsishandsome) [reply tikv-client/src/test/java/com/pingcap/tikv/codec/TableCodecTest.java]: agree

**Pull request merged**